packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -174,6 +178,54 @@ function getStylesDeclarations( blockStyles = {} ) {
 						) }`
 					);
 				} );
+			} else if ( !! properties && isString( styleValue ) && rootOnly ) {
+				const separateValues = styleValue.split( ' ' );
+
+				const sortedBoxValues = {
```

Comment:
What is this default value about?

Reply:
We should probably decide on a fallback value if neither theme nor user provides one; I just used 17px throughout this PR to make debugging easier. But here, I guess it could be 0 or even null, because we always fill in a value for all the properties below.